apps/web/src/components/ChatWindow/ChatInput/ChatInput.tsx

```diff
@@ -32,8 +32,10 @@ const useStyles = makeStyles(theme => ({
     },
   },
   buttonContainer: {
-    margin: '1em 0 0 1em',
+    margin: '1em 0 0 0.5em',
```

Comment:
My only thought here is - should there be 0 left-margin? Just thinking it looks the tiniest bit lop-sided with left-margin and no right-margin. What do you think?
![image](https://user-images.githubusercontent.com/12685223/187090343-d82d274a-cb01-4e52-8170-51f72809c7b7.png)
![image](https://user-images.githubusercontent.com/12685223/187090350-398452aa-3347-4fc9-9408-c696371d1fdc.png)

Reply:
Yep that looks good!